### PR TITLE
New package: qpwgraph-0.0.2

### PR DESCRIPTION
**Pull request by st3r4g**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

Screenshot from 2021-11-06 17-24-31

### CODE REVIEW

**Comment by st3r4g** [comment]
it's still trying to run the target's `moc`...

File: srcpkgs/qpwgraph/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'qpwgraph'
+pkgname=qpwgraph
+version=0.0.2
+revision=1
+wrksrc="${pkgname}-v${version}"
+build_style=cmake
+hostmakedepends="pkg-config qt6-base"
+makedepends="qt6-base-devel pipewire-devel"
```
Comment (ericonr) [inline srcpkgs/qpwgraph/template:8]: Any reason to prefer qt6 over qt5?

File: srcpkgs/qpwgraph/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'qpwgraph'
+pkgname=qpwgraph
+version=0.0.2
```
Comment (ericonr) [inline srcpkgs/qpwgraph/template:3]: 0.0.3 is out

**Review by ericonr — COMMENTED** [review]
Does this bring clear improvements over qjackctl already?

File: srcpkgs/qpwgraph/template (hunk 1 above)
Comment (st3r4g) [reply srcpkgs/qpwgraph/template:8]: not really, I would just stick to what the developer uses more often, which I think is qt6. But if there are reasons to prefer qt5, it's ok.

**Comment by st3r4g** [comment]
> Does this bring clear improvements over qjackctl already?

Mainly the ability to work also for video, also quicker to use (one less click then qjackctl, to open the graph from the floating window).

**Comment by ericonr** [comment]
Merged in #35076, thanks!

**Pull request closed without merging**